Formula/libcanberra.rb

```diff
@@ -7,6 +7,7 @@ class Libcanberra < Formula
   head "git://git.0pointer.de/libcanberra"
 
   depends_on "pkg-config" => :build
+  depends_on "libtool"
```

Comment:
This will raise an audit failure unless you tag it `=> :run`. Should have pointed that out in my initial comment, apologies.